example/dice/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// dice is the "Roll the dice" getting started example application.
```

Comment:
```suggestion
// Package main provides a "Roll the dice" getting started example application.
```

Reply:
Commands (`package main`) have different convention for comments. See: https://go.dev/doc/comment

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/4539/commits/30cd9e8f9512ca885cc66b428dc8bda5f601ace9